lib/pavilion/plugins/commands/file.py

```diff
@@ -0,0 +1,65 @@
+import errno
+import os
+import sys
+
+from pavilion import commands
+from pavilion import utils
+
+class FileCommand(commands.Command):
+
+    def __init__(self):
+        super().__init__(
+            'file',
```

Comment:
Let's call this the 'cat' command, and have an 'ls' and 'file' alias for it.

Reply:
I wonder if this should just be split into two separate plugin commands, e.g., `commands/cat.py` and `commands/ls.py`?